include/openmc/random_lcg.h

```diff
@@ -10,13 +10,11 @@ namespace openmc {
 // Module constants.
 //==============================================================================
 
-constexpr int N_STREAMS         {6};
+constexpr int N_STREAMS         {4};
 constexpr int STREAM_TRACKING   {0};
-constexpr int STREAM_TALLIES    {1};
-constexpr int STREAM_SOURCE     {2};
-constexpr int STREAM_URR_PTABLE {3};
-constexpr int STREAM_VOLUME     {4};
-constexpr int STREAM_PHOTON     {5};
+constexpr int STREAM_SOURCE     {1};
+constexpr int STREAM_URR_PTABLE {2};
+constexpr int STREAM_VOLUME     {3};
```

Comment:
The original idea was that if you run both a volume calculation and a normal calculation, you wouldn't end up reusing the same random numbers for both. However, the only way you could do this right now is through the C API, and even then it probably doesn't matter if you were to reuse the random numbers since they are being used for different purposes. If you'd like, we could remove this stream and just use the tracking one.

Reply:
Ah, yes, seems useful to keep the separate stream if there is a possibility someone may want to perform a volume computation and a regular transport simulation in one go.